hedera-node/docs/system-accounts-operations.md

```diff
@@ -0,0 +1,184 @@
+# System accounts
+
+## Definition
+The Hedera network reserves the first 1000 entity numbers for its own uses, and any account with a number in this reserved range is referred to as a system account.
+
+The following addresses are reserved for the Hedera precompiled contracts:
```

Comment:
Please change the following 3 reserved address to not system not precompile.
The only precompiles are the EVM precompiles
```suggestion
The following addresses are reserved for the Hedera system contracts:
```

Reply:
Done.